po/de.po

```diff
@@ -424,92 +422,88 @@ msgstr "Rei_henfolge umkehren"
 
 #: data/menu.ui.h:19
 msgid "_Split Pages"
-msgstr "Seiten _teilen"
+msgstr "_Seiten zuschneiden"
 
 #: data/menu.ui.h:20
-msgid "_Grid Split"
-msgstr "_Raster zuschneiden"
-
-#: data/menu.ui.h:21
 msgid "Cu_t"
 msgstr "_Ausschneiden"
 
-#: data/menu.ui.h:22
+#: data/menu.ui.h:21
 msgid "_Copy"
 msgstr "_Kopieren"
 
-#: data/menu.ui.h:23
+#: data/menu.ui.h:22
 msgid "Paste _After"
-msgstr "Einfügen _nach Auswahl"
+msgstr "Einfügen _nach"
 
-#: data/menu.ui.h:24
+#: data/menu.ui.h:23
 msgid "Paste _Before"
-msgstr "Einfügen _vor Auswahl"
+msgstr "Einfügen _vor"
 
-#: data/menu.ui.h:25
+#: data/menu.ui.h:24
 msgid "Paste Interleave _Odd"
-msgstr "Abwechselnd einfügen _ungerade"
+msgstr "Leere _ungerade Seite einfügen"
```

Comment:
"Abwechselnd einfügen" is not great, but I think you misunderstood the meaning here: We are not inserting empty pages but pasting from clipboard in a way that alternates an existing page and a pasted page just like the teeth in a zipper.